gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.spec.ts

```diff
@@ -320,6 +324,33 @@ describe('PortalSettingsComponent', () => {
 
       expect(fixture.nativeElement.querySelector('.portal__form_card')).toBeNull();
     });
+
+    it('applies to both portals is present', async () => {
+      const appliesToBothPortals = 'Applies to both portals';
+      portalSettingsMock = fakePortalSettings({
+        portalNext: {
+          access: { enabled: true },
+        },
+      });
+      expectPortalSettingsGetRequest(portalSettingsMock);
+      expect(await componentHarness.getBadgeWarningText()).toEqual(appliesToBothPortals);
+      expect(await componentHarness.getBadgeIconName()).toBeTruthy();
```

Comment:
I don't think this "expect" is relevant.
By checking the icon, I would expect the name of the icon to be the right one.
Here we are just verifying that there is an icon.

Please could you either remove the expectation on the badge icon **or** check that the icon is the right one (by checking the name for instance).

Reply:
updated